### PR TITLE
Update the pcr performance section

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-11079

This updates the PCR performance section and removes some of the previous content.

### CODE REVIEW

File: src/current/v24.1/physical-cluster-replication-overview.md [hunk 1]
```diff
@@ -43,14 +43,10 @@ You can use PCR in a disaster recovery plan to:
...
 
-Cockroach Labs supports PCR up to the following scale:
-
-- Writes: 10,000 writes per second
-- Reads: 18,000 reads per second
-
-Cockroach Labs' testing with [TPC-C]({% link {{ page.version.version }}/cockroach-workload.md %}#tpcc-workload) has demonstrated support for workloads up to the following scale:
+Cockroach Labs testing has demonstrated the following results for workloads up to the outlined scale:
 
 - Initial data load: 500,000 warehouses
```
Comment (alicia-l2) [inline src/current/v24.1/physical-cluster-replication-overview.md:48]: Could we add the "Initial data load: 30TB"

**Review by alicia-l2 — COMMENTED** [review]
lgtm

File: src/current/v24.1/physical-cluster-replication-overview.md (hunk 1 above)
Comment (kathancox) [inline src/current/v24.1/physical-cluster-replication-overview.md:48]:
```suggestion
- Initial data load: 30TB
```

File: src/current/v24.2/physical-cluster-replication-overview.md [hunk 2]
```diff
@@ -42,14 +42,10 @@ You can use PCR in a disaster recovery plan to:
...
 
-Cockroach Labs supports PCR up to the following scale:
-
-- Writes: 10,000 writes per second
-- Reads: 18,000 reads per second
-
-Cockroach Labs' testing with [TPC-C]({% link {{ page.version.version }}/cockroach-workload.md %}#tpcc-workload) has demonstrated support for workloads up to the following scale:
+Cockroach Labs testing has demonstrated the following results for workloads up to the outlined scale:
 
 - Initial data load: 500,000 warehouses
```
Comment (kathancox) [inline src/current/v24.2/physical-cluster-replication-overview.md:47]:
```suggestion
- Initial data load: 30TB
```

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**